model/src/main/kotlin/utils/PurlUtils.kt

```diff
@@ -31,7 +31,6 @@ import org.ossreviewtoolkit.utils.common.percentEncode
  */
 enum class PurlType(private val value: String) {
     APK("apk"),
-    A_NAME("a-name"),
     BAZEL("bazel"),
```

Comment:
We have a bunch of unused purl types here, so why can it be used as a rationale to remove it?

Reply:
Because it was an unofficial addition that was *exclusively* used by Nexus IQ, and is not part of https://github.com/package-url/purl-spec/blob/master/PURL-TYPES.rst.

Reply:
I've clarified that in the commit message as well.